code/modules/projectiles/ammunition/ballistic/rifle.dm

```diff
@@ -49,3 +49,35 @@
 	name = "40mm rubber shell"
 	desc = "A cased rubber slug. The big brother of the beanbag slug, this thing will knock someone out in one. Doesn't do so great against anyone in armor."
 	projectile_type = /obj/projectile/bullet/shotgun_beanbag/a40mm
+
+/obj/item/ammo_casing/rebar
+	name = "sharpened iron rod"
+	desc = "A Sharpened Iron rod. It's Pointy!"
+	caliber = CALIBER_REBAR
+	icon_state = "rod_sharp"
+	base_icon_state = "rod_sharp"
+	projectile_type = /obj/projectile/bullet/rebar
+
+/obj/item/ammo_casing/rebar/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/caseless, TRUE)
+
+/obj/item/ammo_casing/rebar/update_icon_state()
+	. = ..()
+	icon_state = "[base_icon_state]"
```

Comment:
Why are we doing this?

Reply:
A lot of the ammo related code is taken from harpoons and I didnt want to break something.

Reply:
But why are we doing it?

Reply:
im fine to remove it if you feel its unneeded

Reply:
I think the important thing here is "does anything happen differently if it is removed?"
If not then you should remove this. If yes, that was the answer to Melbert's question.

Reply:
Ok, its removed

Reply:
Update, Had to readd it. It makes the sprite display when on the ground, due to being caseless.